c10/core/GeneratorImpl.cpp

```diff
@@ -8,6 +8,7 @@
 #ifndef _WIN32
 #include <fcntl.h>
 #include <unistd.h>
+#else
```

Comment:
That is a big change? do we really want to add chrono on windows and remove it from non windows?

Reply:
A later Windows branch uses chrono   
```
uint64_t getNonDeterministicRandom(bool is_cuda) {               
  // NOLINTNEXTLINE(cppcoreguidelines-init-variables)                        
  uint64_t s;                      
  if (!is_cuda) {
#ifdef _WIN32                                         
    s = (uint64_t)std::chrono::high_resolution_clock::now()
            .time_since_epoch()
            .count();
#elif defined(__SGX_ENABLED__)
    TORCH_CHECK(                                               
        sgx_read_rand(reinterpret_cast<uint8_t*>(&s), sizeof(s)) == SGX_SUCCESS,
        "Could not generate random number with sgx_read_rand.");
#else                             
    s = readURandomLong();
#endif                                                                              
```

Reply:
Ho wow :o That sounds bad...